reproducibility_project/spe_subproject/summarize.py

```diff
@@ -0,0 +1,162 @@
+"""Summarize spe data and save as csv files."""
+import os
+
+import numpy as np
+import pandas as pd
+import signac
+import unyt as u
+
+
+def K_to_kj_per_mol_conversion(value):
+    """Convert energy unit from K to kj/mol."""
+    conversion_factor = (1 * u.Kelvin).to_value(
+        "kJ/mol", equivalence="thermal"
+    )  # conversion_K_to_kj_per_mol
+    converted = value * conversion_factor
+    return converted
+
+
+if __name__ == "__main__":
+    project = signac.get_project()
+
+    engines = ["lammps-VU", "hoomd", "gromacs", "mcccs", "gomc", "cassandra"]
+
+    molecules = [
+        "methaneUA",
+        "pentaneUA",
+        "benzeneUA",
+        "waterSPCE",
+        "ethanolAA",
+    ]
+    properties = [
+        "potential_energy",
+        "tot_vdw_energy",
+        "tail_energy",
+        "tot_electrostatics",
+        "short_range_electrostatics",
+        "long_range_electrostatics",
+        "tot_pair_energy",
+        "bonds_energy",
+        "angles_energy",
+        "dihedrals_energy",
+        "tot_bonded_energy",
+        "intramolecular_energy",
+        "intermolecular_energy",
+    ]
+
+    spe_data = dict()
+    for molecule in molecules:
+        spe_data[molecule] = dict()
+        for engine in engines:
+            spe_data[molecule][engine] = dict()
+            for property in properties:
+                spe_data[molecule][engine][property] = 0
+
+    # Parse LAMMPS SPE data
+    lmp_map = {
+        "potential_energy": "potential_energy",
+        "vdw_energy": "tot_vdw_energy",
+        "coul_energy": "tot_electrostatics",
+        "pair_energy": "tot_pair_energy",
+        "bonds_energy": "bonds_energy",
+        "angles_energy": "angles_energy",
+        "dihedrals_energy": "dihedrals_energy",
+        "tail_energy": "tail_energy",
+        "kspace_energy": "long_range_electrostatics",
+    }
+
+    for job in project.find_jobs({"engine": "lammps-VU"}):
+        data = np.genfromtxt(f"{job.ws}/log-spe.txt", names=True, delimiter=",")
+        for prop in data.dtype.names:
+            spe_data[job.sp.molecule][job.sp.engine][lmp_map[prop]] += data[
+                prop
+            ]
+
+    # Parse HOOMD SPE data
+    hoomd_map = {
+        "pair_LJ": "tot_pair_energy",
+        "pair_LJ_tail": "tail_energy",
+        "pair_Ewald": "short_range_electrostatics",
+        "pppm_Coulomb": "long_range_electrostatics",
+        "special_pair_LJ": "tot_pair_energy",
+        "special_pair_Coulomb": "short_range_electrostatics",
+        "bond_Harmonic": "bonds_energy",
+        "angle_Harmonic": "angles_energy",
+        "dihedral_OPLS": "dihedrals_energy",
+        "potential_energy": "potential_energy",
+    }
+    for job in project.find_jobs({"engine": "hoomd"}):
+        data = np.genfromtxt(
+            f"{job.ws}/log-spe-raw.txt", names=True, delimiter=" "
```

Comment:
So, I'm a bit out of the loop on the details of this project, but poking around `reproducibility_study/reproducibility_project/spe_subproject/workspace`, the jobs with `log-spe-raw.txt` seem to only have data for `pair_LJ pair_LJ_tail potential_energy` .  Maybe that is all that is needed for this sub project?

Other than that, the Hoomd mapping looks good to me.

Reply:
There's supposed to be the log-spe.txt, if you look at the hoomd project.py there's a clear step to write it from log-spe-raw.txt. I wonder how Co was able to access that data. @daico007, can you clarify on this?